tests/src/test/scala/cats/tests/AlgebraInvariantSuite.scala

```diff
@@ -230,11 +270,12 @@ class AlgebraInvariantSuite extends CatsSuite {
   checkAll("Invariant[Monoid]", InvariantTests[Monoid].invariant[Option[MiniInt], Boolean, Boolean])
   checkAll("Invariant[Monoid]", SerializableTests.serializable(Invariant[Monoid]))
 
-  Eq[Band[Set[Boolean]]]
-  cats.laws.discipline.ExhaustiveCheck[Set[Boolean]]
-  Eq[(Set[Boolean], Boolean)]
-  Eq[(Set[Boolean], Set[Boolean] => (Set[Boolean], Boolean))]
-  Eq[CommutativeSemigroup[Set[Boolean]]]
+  //do we need these?
```

Comment:
As per comment, are these lines necessary for anything? The code compiles fine without them.